recOrder/cli/apply_inverse_transfer_function.py

```diff
@@ -23,6 +23,7 @@
     transfer_function_dirpath,
     ram_multiplier,
     unique_id,
+    verbose
```

Comment:
I'm guessing you forgot to add `parsing.py` to the commit? The test show this import is failing.